roles/hive/defaults/main.yaml

```diff
@@ -0,0 +1,165 @@
+# Used to cleanup the hive installation
+hive_cleanup: false
+
+hive_repo_address: "https://github.com/ethereum/hive.git"
+hive_repo_version: "master"
+hive_repo_remote: "origin"
+
+hive_systemd_service_name: "hive.service"
+
+hive_dir: "/data/hive"
+hive_dir_bin: "{{ hive_dir }}/bin"
+hive_dir_source: "{{ hive_dir }}/src"
+hive_dir_scripts: "{{ hive_dir }}/scripts"
+hive_dir_results: "{{ hive_dir }}/results"
+hive_dir_web: "{{ hive_dir }}/web"
+
+hive_container_www_name: "nginx-hive"
+hive_container_www_image: "nginx:1.27.3-alpine"
+hive_container_www_env: {}
+hive_container_www_ports:
+  - "127.0.0.1:8080:80"
+hive_container_www_volumes:
+  - "{{ hive_dir_web }}:/usr/share/nginx/html"
+  - "{{ hive_dir_results }}:/usr/share/nginx/html/results"
+hive_container_www_networks: []
+hive_container_www_restart_policy: "always"
+hive_container_www_stop_timeout: 10
+
+
+hive_simulations:
+  # Always pull / build the latest container images
+  always_pull_images: true
+  # Global flags provided to all test runs
+  global_flags:
+    - --docker.nocache=.
+    - --sim.loglevel=3
+    - --loglevel=3
+  # Exclude these containers from cleanup /
+  container_cleanup_exclusions:
+    - telegraf
+    - nginx
+    - logsprout
+    - papertrail
+    - rsyslog
+    - vector
+    - node_exporter
+    - prometheus
+    - grafana
+  # Prune test results older than this value
+  prune_test_results_older_than: 720h # 30 days
```

Comment:
Might need to be lower? afaik even few days of test results are enough and 30d of tests might be waaayy too big for most hdds. I could be wrong about the number, but we tuned to a sane default on the prod hive instances.

Reply:
Made it default to 7d. 👍